plugins/blocklayer/plugin.go

```diff
@@ -165,9 +166,11 @@ func configure(plugin *node.Plugin) {
 			}
 		}
 
+		activityLogConsumer := func(activityLog epoch.SnapshotEpochActivity) {}
```

Comment:
let's pass nil instead of these empty consumers to highlight the fact that this component is not doing anything with this part of the data. The `LoadSnapshot` should support this of course; this is a comment I've already made to @jkrvivian, so it might be already implemented in her PR.